app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_a_user_posts_counter
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [ ] Good job working on this method, but it will be nicer if you implemented a Rails Active Callback `after_save` to call the `update_post_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private. This will update the count every time a new post is saved.

```ruby
# app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```

Reply:
Thank you for the descriptive feedback. Done